common/src/main/java/org/jboss/da/common/util/LoggerProducer.java

```diff
@@ -16,4 +16,11 @@
     public Logger produceLogger(InjectionPoint ip) {
         return LoggerFactory.getLogger(ip.getMember().getDeclaringClass());
     }
+
+    @Produces
+    @UserLog
+    public Logger produceUserLogger() {
+        return LoggerFactory.getLogger("org.jboss.pnc.userlog.da");
```

Comment:
Logger name have to be "org.jboss.pnc.\_userlog\_.da"

Reply:
JIRA understoot the underscores as formating for italics :/